frontend/src/Routes/DynamicRoute/DynamicRoute.tsx

```diff
@@ -57,21 +43,26 @@ const DynamicRoute: React.FC<DynamicRouteProps> = ({ location }) => {
               };
             }
           }),
-        );
-      }
-    }
-  }, [location, dynamicRoutePages]);
+        };
+      }),
+    [dynamicRoutePages],
+  );
 
   return (
-    <section className={currClassName || ''}>
-      {Component ? (
-        <React.Suspense fallback={null}>
-          <Component />
-        </React.Suspense>
-      ) : (
-        <Loader />
-      )}
-    </section>
+    <React.Suspense fallback={null}>
+      {routes.map(({ className, Component, uid, ...currCoute }) => (
+        <React.Fragment key={uid}>
+          <Route
+            {...currCoute}
+            render={() => (
+              <section className={className}>
```

Comment:
Won't having this intermediate `section` prevent pages from being able to create content that consumes the full height of the page content? In most cases it's probably ok because the content wants the page to own the scrolling. 
But in other cases the content may want to consume the full height and control their own layout. But now the content will be confined within a block element with no defined height.

Reply:
Ah good point, let's use `article` instead of `section` so we don't use a block element.

Reply:
Although, `article` is `block` element as well. Mhh, this is not configurable from the app side as well (since we do wrap each CSS rule with the app name). I think it's fine for now and let's keep an eye on future requests. There's already an issue opened to fix the problematic namespacing of all CSS rules and not an option to style parent components.

Reply:
My point wasn't about a block or inline element but the fact that there is an intermediate node inserted that affects the ability of the child node to grow to the size of the content container through css.

It's fine for now, but be aware that this can affect page layouts that want to consume all available space.